libzeth/test/commitments_test.cpp

```diff
@@ -189,9 +189,11 @@ TEST(TestCOMMs, TestCOMMGadget)
     bits256 a_pk_bits256 = get_bits256_from_vector(
         hex_digest_to_binary_vector("5c36fea42b82800d74304aa4f875142b42"
                                     "1b4f2847e7c41c1077fbbcfd63f886"));
-    bits256 cm_bits256 = get_bits256_from_vector(
-        hex_digest_to_binary_vector("f1378e66b0037337743d1ca5c83ed28c4e"
-                                    "873c3fb242dcef3ff0db4ebe82c15f"));
+    // bits256 cm_bits256 = get_bits256_from_vector(
+    //    hex_digest_to_binary_vector("f1378e66b0037337743d1ca5c83ed28c4e"
+    //                                "873c3fb242dcef3ff0db4ebe82c15f"));
```

Comment:
Shall we remove these comments? Otherwise let's use the `cm_bits256` variable to build the input of the `FieldT` constructor below

Reply:
Updated.